### PR TITLE
Get dhclient_program from jail config

### PR DESCRIPTION
**Pull request by mekanix**

### CODE REVIEW

**Comment by mekanix** [comment]
I didn't want to merge it right away. I must be mentally tired from isolation, so if you want me, I'll revert this PR. Sorry!

File: sudoexec/jstop [hunk 1]
```diff
@@ -213,7 +213,8 @@ sleep 2
 external_exec_script -s stop.d
 if [ ${vnet} -eq 1 ]; then
 	if [ "${ip4_addr}" = "REALDHCP" ]; then
-		${JEXEC_CMD} ${jname} /bin/pkill -9 dhclient
+		${JEXEC_CMD} ${jname} /bin/pkill -9 dhclient 2>/dev/null 1>&2
```
Comment (olevole) [inline sudoexec/jstop:216]: As far as I can see, we can do one pkill call ( pkill pattern ... ), e.g:

${JEXEC_CMD} ${jname} /bin/pkill -9 dhclient dhcpd 2>/dev/null 2>&1

File: sudoexec/jstart [hunk 2]
```diff
@@ -656,7 +656,13 @@ if [ ${vnet} -eq 1 ]; then
 		echo "${jname}" > ${tmpdir}/eth${eth_seq}.locked
 		${IFCONFIG_CMD} ${i} name eth${eth_seq} && ${IFCONFIG_CMD} eth${eth_seq} vnet ${jname}
 		if [ "${ip4_addr}" = "REALDHCP" ]; then
-			${JEXEC_CMD} ${jname} /sbin/dhclient eth${eth_seq}
+			. "${workdir}/jails-data/${jname}-data/etc/rc.conf.d/network" 2>/dev/null 1>&2
```
Comment (olevole) [inline sudoexec/jstart:659]: mb test for file existance? 

[ -r ${workdir}/jails-data/${jname}-data/etc/rc.conf.d/network ] && . ${workdir}/jails-data/${jname}-data/etc/rc.conf.d/network

?

**Comment by olevole** [comment]
> I didn't want to merge it right away. I must be mentally tired from isolation, so if you want me, I'll revert this PR. Sorry!

Someday this isolation going to end !

**Pull request merged**